.gitignore

```diff
@@ -46,6 +46,7 @@ win32/BuildErrors.txt
 win32/BuildErrors-64bit.txt
 win32/.vs
 win32/tsk-win.VC.VC.opendb
+win32/tsk-win.VC.opendb
```

Comment:
Is the preceding line a typo that needs to be removed?

Reply:
No, we need both. One of these files is used in Visual Studio as a cache, and it depends on the version. It may cause permission error if uploaded.

(Reference: https://stackoverflow.com/questions/34975423/visual-studio-2015-git-error-opensomefile-vc-opendb-permission-denied-fa)